.golangci-maximal.yml

```diff
@@ -0,0 +1,182 @@
+linters:
+  disable-all: true
+  enable:
+    - copyloopvar
+    - cyclop
```

Comment:
I'd recommend skipping tests for `cyclop`.

```yml
linters-settings:
  cyclop:
    skip-tests: true
```

Reply:
I do think it's worth holding test code to same standards of readability we'd apply to other code. Is there a particular reason you think this should be skipped for test code?

Reply:
That's a good point, readability of test code is critical to useful tests. I made this suggestion after running the linters on the Go Driver repo and looking for what I thought were false positives and overly noisy ones, but perhaps we should look at our test code complexity instead 😅. Disregard this suggestion.